src/main/scala/com/monsanto/arch/cloudformation/model/simple/Builders.scala

```diff
@@ -203,18 +203,20 @@ trait Autoscaling {
     userData:     `Fn::Base64`,
     iam:          Option[Token[ResourceRef[`AWS::IAM::InstanceProfile`]]] = None,
     condition:    Option[ConditionRef] = None,
-    dependsOn:    Option[Seq[String]]  = None
+    dependsOn:    Option[Seq[String]]  = None,
+    blockDevices: Option[Seq[BlockDeviceMapping]] = None
```

Comment:
@ddgenome I can change it if you want, but I do know that microservices-vpc doesn't name parameters when it uses this, so it'd need to be updated when we move to this release of cftg.  Let me know your preference.